lib/usd/ui/layerEditor/layerTreeItemDelegate.cpp

```diff
@@ -283,7 +289,11 @@ void LayerTreeItemDelegate::paint_ActionIcon(
 
     QString tooltip;
     if (_lastHitAction == action._name) {
-        tooltip = action._tooltip;
+        if (item->isSystemLocked()) {
```

Comment:
When a layer is system locked, a different message is shown to the user to indicate that a layer cannot be unlocked or edited.